tests/foreman/api/test_docker.py

```diff
@@ -108,7 +112,7 @@ def _create_repository(product, name=None, upstream_name=None):
 
     """
     if name is None:
-        name = gen_string(gen_choice(STRING_TYPES), 15)
+        name = generate_strings_list(15, ['numeric', 'html'])
```

Comment:
I just wanted to introduce only equal changes as the point of this pr is performance improvement, not any other kind of fix. It will take extra effort to verify all the possible data types and whether they are working or not, and we don't have much time as we should still focus on remaining test failures. And, basically, right now i'm 99% sure that we can go ahead and merge this pull request, but if i'll introduce some other changes then i'll definitely need to test everything.